### DISCUSSION TITLE
賢者のアミュレットの生成階層を浅くして欲しい

### DISCUSSION THREAD

**Discussion by daradarach** [hengband/hengband#220] [Closed] [answered]
中盤で拾ってもイマイチ使えない賢者のアミュレットですが、
短愚のように浅いところから生成されれば、
弱テレパシーとしてよし、売ってよしの便利アイテムとなるのでもう少し浅いところから生成されて欲しい。

**Comment by deskull-m** [comment]
バランス要素として考慮します。

**Comment by iks3** [comment, marked as answer]
以下のトピックに吸収したいと思います。

ベースアイテム調整案について (3.0.0α→3.0.0β)
https://github.com/hengband/hengband/discussions/300
